adapter/prometheus/prometheus_test.go

```diff
@@ -69,7 +69,7 @@ var (
 		Name:        "unknown",
 		Description: "unknown",
 		Kind:        adapter.Gauge - 2,
```

Comment:
The type of the field is MetricKind, given it lives in a MetricDefinition I think it's overkill to call it MetricKind, but I like Kind more than Flavor. We could call the enum MetricType and this field Type, I suppose.